turbinia/jobs/binary_extractor.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Job to execute binary_extractor task."""
+
+from __future__ import unicode_literals
+
+from turbinia.evidence import APFSEncryptedDisk
+from turbinia.evidence import BitlockerDisk
+from turbinia.evidence import Directory
+from turbinia.evidence import GoogleCloudDisk
+from turbinia.evidence import GoogleCloudDiskRawEmbedded
+from turbinia.evidence import RawDisk
+from turbinia.evidence import BinaryExtraction
+from turbinia.jobs import interface
+from turbinia.jobs import manager
+from turbinia.workers.binary_extractor import BinaryExtractorTask
+
+
+class BinaryExtractorJob(interface.TurbiniaJob):
+  """Run image_export.py on evidence to extract binaries."""
+
+  # The types of evidence that this Job will process.
+  evidence_input = [
+      Directory, RawDisk, GoogleCloudDisk, GoogleCloudDiskRawEmbedded,
```

Comment:
If you include the encrypted Evidence types here you'll need to also need to use the `recovery_key` and `password` attributes.  See the `workers/plaso.py` for an example.  You'll also need to make sure that `image_export.py` supports these flags, and after a cursory glance it looks like it might not, so you may need to add some functionality there.  If it's not supported yet you might want to just remove those as possible inputs.

Reply:
I will remove ecrypted evidence for now, since image_export.py does not support these flags, but will get back to this once I fix that.

Reply:
SGTM.